### ISSUE TITLE
 [You're a Mean One...]  Winter Veil event

### ISSUE THREAD

**Issue by deni6** [WoWManiaUK/Blackwing-Lair#4957]
**https://www.wowhead.com/quest=7043/youre-a-mean-one**

**What is happening:**
First of all the boss has 144M hp and it takes almost 1h to kill him. (5men) 
After boss die the reindeer is set free you get the credit (on the middle of the screen 1/1) but on the quest 
the reindeer is still 0/1 

**What should happen:**
Reindeer shall be set free after boss die (grinch) and quest should be finished 

**Is it crashing the server?:** 
No

**Other Information:**
Please add the q on .bq or if you can fix it fast ok (due to this is world event with limited time)

**Comment by Echinaceax** [comment]
yes we tried to get all Winter veil fixed in time, but there maybe some issues sadly @deni6

**Comment by Echinaceax** [comment]
We did it with 10 man 
one thing we did notice tho was not only the health but the boss does no dmg ...didnt report that part i see 

as for the quest, it is completable - Just visual bug on the quest log 

![on metzen](https://user-images.githubusercontent.com/39439201/102234218-5c35c400-3ef1-11eb-84df-343ac61f328a.jpg)

**Comment by Blametheheal** [comment]
https://cata-twinhead.twinstar.cz/?npc=13602
His health is only supposed to be 3,435,680

**Comment by Echinaceax** [comment]
Also fix this too while your on this NPC @mfranca0009 Please :)
#4961
